guides/maintainer/README.md

```diff
@@ -0,0 +1,13 @@
+# Maintainers Guide
+
+Welcome! This guidebook is designed to collect and share best practices,
+processes, and other information on being a maintainer of an OpenTelemetry
+project.
+
+## Table of Contents
+
+TODO
```

Comment:
I'd like to see something about "set a good example". Say, if there is an issue template out there, even if you feel like it is a burden, use it or fix it so it is not burdensome. "remember your roots" Getting to approver status may have felt uphill for you, remember that and actively seek new approvers possibly on a schedule so that you don't forget (e.g. sweep every quarter). Enough suggestions for now!

Reply:
going to merge this now, raise a follow up issue for further discussion